Yona/Yona/Friends/AddFriendsViewController.swift

```diff
@@ -306,7 +306,7 @@ class AddFriendsViewController: UIViewController, UIScrollViewDelegate, UINaviga
                 let trimmedWhiteSpaceString = number.removeWhitespace()
                 let trimmedString = trimmedWhiteSpaceString.removeBrackets()
                 
-                if trimmedString.validateMobileNumber() == false {
+                if !trimmedString.isValidMobileNumber(){
```

Comment:
Change the lines 300 till this one into something like this:
```
            if let mobilenum = mobileTextfield.text {
                let number = formatNumber(mobilePrefixTextField.text, mobileNum)
                
                if !number.isValidMobileNumber(){
```

Inside ``formatNumber``, do the following:
* If the prefix is not empty and the number does not start with a ``+` yet, prefix that to the number
* Apply ``removeWhitespace``
* Apply ``removeBrackets``
* Apply ``formatDutchCountryCodePrefix``

Reply:
Updated accordingly code.